bridges/sbus_bridge/fc_settings/lumenier_f4_aio_betaflight-v3.2.5_rpg_race_quad.txt

```diff
@@ -0,0 +1,663 @@
+set osd_home_dir_pos = 302
+set osd_home_dist_pos = 303
+set osd_compass_bar_pos = 266
+set osd_altitude_pos = 161
+set osd_pid_roll_pos = 322
+set osd_pid_pitch_pos = 354
+set osd_pid_yaw_pos = 386
+set osd_debug_pos = 1
+set osd_power_pos = 47
+set osd_pidrate_profile_pos = 418
+set osd_warnings_pos = 329
+set osd_avg_cell_voltage_pos = 2145
+set osd_pit_ang_pos = 257
+set osd_rol_ang_pos = 289
+set osd_battery_usage_pos = 392
+set osd_disarmed_pos = 138
+set osd_nheading_pos = 311
+set osd_nvario_pos = 279
+set osd_esc_tmp_pos = 82
+set osd_esc_rpm_pos = 83
+set osd_stat_max_spd = ON
+set osd_stat_max_dist = OFF
+set osd_stat_min_batt = ON
+set osd_stat_min_rssi = ON
+set osd_stat_max_curr = ON
+set osd_stat_used_mah = ON
+set osd_stat_max_alt = OFF
+set osd_stat_bbox = ON
+set osd_stat_endbatt = OFF
+set osd_stat_bb_no = ON
+set osd_stat_tim_1 = OFF
+set osd_stat_tim_2 = ON
+set task_statistics = ON
+set debug_mode = NONE
+set cpu_overclock = OFF
+set pwr_on_arm_grace = 5
+set vtx_halfduplex = ON
+set vcd_video_system = 0
+set vcd_h_offset = 0
+set vcd_v_offset = 0
+set max7456_clock = FULL
+set displayport_msp_col_adjust = 0
+set displayport_msp_row_adjust = 0
+set displayport_max7456_col_adjust = 0
+set displayport_max7456_row_adjust = 0
+set displayport_max7456_inv = OFF
+set displayport_max7456_blk = 0
+set displayport_max7456_wht = 2
+set esc_sensor_halfduplex = OFF
+set led_inversion = 0
+set dashboard_i2c_bus = 1
+set dashboard_i2c_addr = 60
+set camera_control_mode = HARDWARE_PWM
+set camera_control_ref_voltage = 330
+set camera_control_key_delay = 180
+set camera_control_internal_resistance = 470
+
+# profile
+profile 0
+
+set dterm_lowpass_type = BIQUAD
+set dterm_lowpass = 100
+set dterm_notch_hz = 260
+set dterm_notch_cutoff = 160
+set vbat_pid_gain = OFF
+set pid_at_min_throttle = ON
+set anti_gravity_threshold = 350
+set anti_gravity_gain = 1000
+set setpoint_relax_ratio = 100
+set dterm_setpoint_weight = 0
+set acc_limit_yaw = 100
+set acc_limit = 0
+set crash_dthreshold = 50
+set crash_gthreshold = 400
+set crash_setpoint_threshold = 350
+set crash_time = 500
+set crash_delay = 0
+set crash_recovery_angle = 10
+set crash_recovery_rate = 100
+set crash_limit_yaw = 200
+set crash_recovery = OFF
+set iterm_windup = 50
+set iterm_limit = 150
+set pidsum_limit = 500
+set pidsum_limit_yaw = 400
+set yaw_lowpass = 0
+set p_pitch = 58
+set i_pitch = 0
+set d_pitch = 35
+set p_roll = 44
+set i_roll = 0
+set d_roll = 30
+set p_yaw = 35
+set i_yaw = 10
+set d_yaw = 20
+set p_alt = 50
+set i_alt = 0
+set d_alt = 0
+set p_level = 50
+set i_level = 50
+set d_level = 100
+set p_vel = 55
+set i_vel = 55
+set d_vel = 75
+set level_limit = 55
+set horizon_tilt_effect = 75
+set horizon_tilt_expert_mode = OFF
+set gps_pos_p = 15
+set gps_pos_i = 0
+set gps_pos_d = 0
+set gps_posr_p = 34
+set gps_posr_i = 14
+set gps_posr_d = 53
+set gps_nav_p = 25
+set gps_nav_i = 33
+set gps_nav_d = 83
+
+# rateprofile
+rateprofile 0
+
+set rc_rate = 213
```

Comment:
Why are these values different from version 3.1.7? These rate values must correspond to what is set in the sbus bridge parameters! -> e.g. https://github.com/uzh-rpg/rpg_quadrotor_control/blob/7381dc715e8414a4e83f67ef87e84857045acdca/bridges/sbus_bridge/parameters/race_quad.yaml#L17